README/ReleaseNotes/v622/index.md

```diff
@@ -190,6 +196,9 @@ See [core/sanitizer](https://github.com/root-project/root/tree/master/core/sanit
   and skipping the file, it now throws an exception if any of the input files is unreadable (this could also happen in
   the middle of an event loop). See [ROOT-10549](https://sft.its.cern.ch/jira/browse/ROOT-10549) for more details.
 - New analysis examples based on the recent ATLAS Open Data release ([`Higgs to two photons`](https://root.cern/doc/master/df104__HiggsToTwoPhotons_8py.html), [`W boson analysis`](https://root.cern/doc/master/df105__WBosonAnalysis_8py.html), [`Higgs to four leptons`](https://root.cern/doc/master/df106__HiggsToFourLeptons_8py.html))
+- An exception is now thrown in case the size of ROOT's thread-pool changes between RDataFrame construction time and the time the event loop begins.
+- Just-in-time compilation of large portions of the computation graph has been optimized, and it is now much faster. Please report any regressions you might encounter.
+- `MakeRootDataFrame` is now a safe way to construct RDFs. It used to return RDFs with more limited functionality.
```

Comment:
```suggestion
- `MakeRootDataFrame` now constructs more reliable RDataFrames. It used to return RDFs with limited functionality.
```

Reply:
I like my stronger wording better sorry :smile: If people were using `MakeRootDataFrame` in their code, they should be scared.